cookbook/recipes/nodejs.rb

```diff
@@ -20,13 +20,13 @@
 
 apt_repository 'node_0.12' do
   uri 'https://deb.nodesource.com/node_10.x/'
-  distribution 'xenial' #node['lsb']['codename']
```

Comment:
why this upgrade?

Reply:
version 10.23.0 is listed as a bionic distribution here, not xenial https://www.ubuntuupdates.org/pm/nodejs

I am not certain if this is going to fix the issue, but out of other ideas

Reply:
will this break older production instances?